fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/postgresql/PostgreSqlAdapter.java

```diff
@@ -150,7 +150,22 @@ public void dropType(String schemaName, String typeName) {
 
     @Override
     public void createOrReplaceProcedure(String schemaName, String procedureName, Supplier<String> supplier) {
-        warnOnce(MessageKey.CREATE_PROC, "Create procedure not supported in PostgreSql");
+        final String objectName = DataDefinitionUtil.getQualifiedName(schemaName, procedureName);
+        logger.info("Create or replace procedure " + objectName);
+
+        // Build the create procedure DDL and apply it
+        final StringBuilder ddl = new StringBuilder();
+        ddl.append("CREATE OR REPLACE FUNCTION ");
+        ddl.append(objectName);
+        ddl.append(System.lineSeparator());
+        ddl.append(supplier.get());
```

Comment:
what does the line separator get you, a pretty print?

Reply:
this is interesting, just followed the existing pattern, let me change this.